src/components/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import Settings from "./Settings";
+
+const Sidebar = () => {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  return (
+    <>
+      {showSidebar ? (
+        <button
+          className="flex text-3xl text-white items-center cursor-pointer fixed right-10 top-6 z-50 right-5"
+          onClick={() => setShowSidebar(!showSidebar)}
+        >
+          <XMarkIcon className="w-12 fill-warning-dark flex-none stroke-white" />
```

Comment:
For icons, I would include an `aria-label` for accessibility (indicates what the icon does)